### PR TITLE
USWDS-Sandbox - Zombie test form: Replace form links with buttons

### LINKED ISSUE #113: Zombie prototype form - Update next step buttons
## Description

Buttons within forms should be actual buttons and not links styled as buttons as stated by our [button guidance](https://designsystem.digital.gov/components/button/).

Update form links to be buttons

> **Warning**
> This should be completed after #106 is merged to avoid conflict

## Related issue

Related to Usability testing [findings](https://github.com/uswds/uswds/issues/5527)

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Replace next step links with actual buttons. 

Change made a result of our Usability testing [findings](https://github.com/uswds/uswds/issues/5527).

## Related issue

Closes #113

## Related pull requests

> **Warning**
Work should be merged after #103
> 

## Preview link

[Zombie test form →](https://federalist-d5e7c07c-6ffa-4b8e-935f-49a7df24a505.sites.pages.cloud.gov/preview/uswds/uswds-sandbox/cm-zombie-defense-update-buttons/)

## Problem statement

Buttons within forms should be actual buttons and not links styled as buttons as stated by our [button guidance](https://designsystem.digital.gov/components/button/).

## Solution

Replace all form links styled as buttons with actual buttons. 

Buttons navigate to new pages via `onclick="location.href = '/page-destination'`

## Testing and review

1. Navigate through test form with and without screen reader.
2. Confirm the next and previous page buttons work as expected.
3. Confirm screen reader callouts are accurate.
4. Confirm screen readers are able to target the buttons using button shortcuts.
    - There doesn’t appear to be a shortcut for VoiceOver
    - For JAWs and NVDA this is `b` **********(forwards)********** or `shift + b` ***********(backwards)***********

### Testing checklist

- [ ]  Form navigates as expected
- [ ]  No visual regression
- [ ]  Screen readers call out buttons as expected
- [ ]  Screen reader navigation works as expected
- [ ]  Confirm this is the proper method to navigate through a form with buttons

### CODE REVIEW

**Review by amycole501 — COMMENTED** [review]
The "Sign in" button is now coded as a button so that's awesome! The "show/hide password" is styled as a link but it's functionally a button as is the "here's how you know" "button" at the top of the page in the banner. Both look like links but are actually buttons. Could prove confusing to someone who is using a screen reader to find all the links on a page and instead find a button.

File: src/confirm-submission.html [hunk 1]
```diff
@@ -32,8 +77,8 @@ <h1 class="margin-bottom-5">Step 4 of 4: Confirm submission</h1>
               </p>
             </section>
 
-            <a href="{{ '/instructor-details' | url }}" class="usa-button usa-button--outline margin-y-4" type="submit">Go back to instructor details</a>
-            <a href="{{ '/' | url }}" class="usa-button margin-y-4" type="submit">Submit</a>
+            <button onclick="location.href = '/instructor-details'" class="usa-button usa-button--outline margin-y-4" type="submit">Go back to instructor details</button>
```
Comment (mejiaj) [inline src/confirm-submission.html:80]: @mahoneycm this fails in preview, but you could use something like `window.location.host` to ensure the right base URL gets used in all environments.

**Steps to reproduce**
1. Go to [homepage](https://federalist-d5e7c07c-6ffa-4b8e-935f-49a7df24a505.sites.pages.cloud.gov/preview/uswds/uswds-sandbox/cm-zombie-defense-update-buttons/).
2. Click `Sign in` button.
3. Cloud pages 404 appears.

**Review by mejiaj — CHANGES_REQUESTED** [review]
@mahoneycm none of the links seem to work in cloud pages previews. Added comment for possible fix.

**Comment by mahoneycm** [comment]
@mejiaj  thanks for catching that! I was able to resolve by returning the [11ty URL filter](https://www.11ty.dev/docs/filters/url/).

I considered using the `<path>` plugin and prefix it mentions on that page, but since the branch name will change I figured this is a better pattern for now.

**Review by mejiaj — APPROVED** [review]
Good fix. Confirming all links work in preview.

@mahoneycm can you get a PR approval from either @amycole501 or @alex-hull? We can merge after.

**Review by amycole501 — APPROVED** [review]

**Pull request merged**